cxx-sensors/src/main/java/org/sonar/cxx/sensors/drmemory/CxxDrMemorySensor.java

```diff
@@ -85,15 +91,6 @@ protected void processReport(final SensorContext context, File report) {
           }
         }
       }
-    } catch (IOException e) {
```

Comment:
The only IO operations in this scope are handled with a try..catch block in ```org.sonar.cxx.sensors.utils.normalizePath(String filename)``` and ```getElements(file, charset)```.

I have also some doubts with the validateRecovery in this context. Whenever a report cannot be parsed (IOException) the analysis should not be stopped. This is the current behavior for other tools with reliability issues like Vera++, PC-lint and RATS.